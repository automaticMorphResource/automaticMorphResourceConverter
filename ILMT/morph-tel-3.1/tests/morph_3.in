1	ixi	unk	
1	cAlA	unk	
1	maMxiki	unk	
1	weVliyaxu	unk	
1	.	unk	
