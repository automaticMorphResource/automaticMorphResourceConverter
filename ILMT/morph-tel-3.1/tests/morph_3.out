1	ixi	unk	<fs af='ixi,pn,fn,sg,3,d,0,0'>
2	cAlA	unk	<fs af='cAlA,avy,,,,,0,0_avy'>|<fs af='cAlu,v,any,any,any,,i_A,i_A'>|<fs af='cAlu,avy,,,,,A,A_avy'>
3	maMxiki	unk	<fs af='maMxi,n,,pl,,,ki,ki'>
4	weVliyaxu	unk	<fs af='weVliyu,v,fn,sg,3,,a,a'>
5	.	unk	<fs af='&dot;,punc,,,,,,'>
